Makefile

```diff
@@ -0,0 +1,51 @@
+GO ?= $(shell command -v go 2> /dev/null)
+MATTERMOST_CLOUD_IMAGE ?= mattermost/mattermost-cloud:test
+MACHINE = $(shell uname -m)
+BUILD_IMAGE = golang:1.12
+BASE_IMAGE = alpine:3.9
+GOPATH ?= $(shell go env GOPATH)
```

Comment:
This is a go-module-enabled repository, so the `GOPATH` should be immaterial. Can you elaborate on why we need to define/trim it below?

Reply:
it is being used to trim the path, if you think it is not right let me know I can remove

Reply:
GOPATH as it exists today will be deprecated in an upcoming Go release, but it sounds like we're just trying to remove the current working directory -- could we trim that instead?